src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -118,16 +121,32 @@
                 <article class="mt-n4">
                   <v-checkbox
                     id="confirm-name-change-chkbx"
-                    class="mb-6"
+                    class="mb-6 sp-confirm-label"
                     :label="`I confirm ${orgPersonLabel} has legally changed their name and that they remain the ` +
-                      `same person.`"
+                      `same business.`"
                     :hide-details="true"
                     :rules="confirmNameChangeRules"
                     v-model="orgPerson.confirmNameChange"
                   />
                 </article>
               </template>
 
+              <!-- Future Firm incorporation number -->
+              <template v-if="orgPerson.officer.identifier">
+                <article class="sp-number-tags">
+                  <label class="sub-header">Incorporation Number:</label>
+                  <span class="sp-incopr-busi-number-text">{{ 'orgPerson.officer.identifier' }}</span>
+                </article>
+              </template>
+
+              <!-- Future Firm business number -->
```

Comment:
This should actually work now, so it's not future.

If you do need to comment something as future work, follow the existing convention so that we can search for this.